public/scripts/extensions/vectors/settings.html

```diff
@@ -178,13 +178,13 @@ <h4>
                 </div>
             </div>
 
-            <h4>
+            <h4 data-i18n="File vectorization settings">
                 File vectorization settings
             </h4>
 
             <label class="checkbox_label" for="vectors_enabled_files">
                 <input id="vectors_enabled_files" type="checkbox" class="checkbox">
-                Enabled for files
+                <div data-i18n="Enable for files">Enable for files</div>
```

Comment:
Same as above - prefer span for inline elements

Reply:
fixed by https://github.com/SillyTavern/SillyTavern/pull/2376/commits/3c696c64b341436768746863591250da6c336932